googlemock/test/gmock-matchers_test.cc

```diff
@@ -2778,7 +2778,7 @@ class FloatingPointTest : public testing::Test {
 
   // Pre-calculated numbers to be used by the tests.
 
-  const size_t max_ulps_;
+  const unsigned int max_ulps_;
```

Comment:
Meaning that you want to change Floating::kMaxUlps to unsigned int too, or meaning that you want to leave both of them as size_t and find another way (a bunch of casts, I guess) to silence the warnings about narrowing from size_t to unsigned int?